f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -665,7 +665,8 @@ def _open_bigip(self, hostname):
             bigip = ManagementRoot(hostname,
                                    self.conf.icontrol_username,
                                    self.conf.icontrol_password,
-                                   timeout=f5const.DEVICE_CONNECTION_TIMEOUT)
+                                   timeout=f5const.DEVICE_CONNECTION_TIMEOUT,
+                                   debug=True)
```

Comment:
We should only set debug=True when the agent INI setting is debug=True. To do so, add a config option to the OPT array in this file, something like:
cfg.BoolOpt(
        'debug',
        default=False,
        help='Enable debug logging.'
    )

You can then pass that value into the ManagementRoot constructor:
...
debug=self.conf.debug)